src/fides/api/ctl/migrations/versions/68a518a3c050_system_managers.py

```diff
@@ -0,0 +1,209 @@
+"""Adds system managers table and client.systems
+
+Also includes a data migration to add new scopes to fidesuserpermission.scopes and client.scopes if they
+have cli-objects:-* scopes as we're making these scopes more granular.
+
+Revision ID: 68a518a3c050
+Revises: 9f38dad37628
+Create Date: 2023-02-23 21:52:56.225405
+
+"""
+from typing import List
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+from sqlalchemy import text
+from sqlalchemy.engine import ResultProxy
+from sqlalchemy.sql.elements import TextClause
+
+revision = "68a518a3c050"
+down_revision = "9f38dad37628"
```

Comment:
I think you'll need to bump this up to `2f6aa5fe797a` after today's merges into `main`

Reply:
yep have to get this up to date again

Reply:
Updated with main - this gets out of date about every day though! imagine we'll need a few more merges before this goes out.